frontend/src/pages/Risk/Risk.tsx

```diff
@@ -344,7 +346,7 @@ const Risk: React.FC = (props) => {
           <h2>{title}</h2>
         </div>
         <div className={cardClasses.body}>
-          {/* <h4 style={{ float: 'left' }}>Today:</h4> */}
+           <ul className={cardClasses.labels}>{labels.map(label => <li>{label}</li>)}</ul>
```

Comment:
Content is a place where we want to avoid being terse. It's best to save the dynamic applications to generating the data structures in a way to make them efficient and accessible. So please hard-code those label names so that we can have a better visual understanding of what we will be rendering.